karapace/constants.py

```diff
@@ -10,6 +10,8 @@
 API_VERSION_AUTO_TIMEOUT_MS: Final = 30000
 TOPIC_CREATION_TIMEOUT_MS: Final = 20000
 DEFAULT_SCHEMA_TOPIC: Final = "_schemas"
+DEFAULT_PRODUCER_MAX_REQUEST: Final = 1048576
+DEFAULT_AIOHTTP_CLIENT_MAX_SIZE = 1024**2
```

Comment:
```suggestion
DEFAULT_AIOHTTP_CLIENT_MAX_SIZE: Final = 1024**2
```

issue: I assume this one should've also received `Final`.

Reply:
It is a bit confusing that those numerical values are equal to each other, but simply represented differently, one as simple value and other as exponential notation.